client/verta/verta/registry/entities/_model.py

```diff
@@ -907,3 +913,23 @@ def delete(self):
         request_url = "{}://{}/api/v1/registry/registered_models/{}".format(self._conn.scheme, self._conn.socket, self.id)
         response = _utils.make_request("DELETE", request_url, self._conn)
         _utils.raise_for_http_error(response)
+
+    def set_action_type(self, action_type):
```

Comment:
These functions should also have docstrings (similar to [lock level](https://github.com/VertaAI/modeldb/blob/96d104b2bf2737989fe729f7828a08fe6bea2bf0/client/verta/verta/registry/entities/_modelversion.py#L1201-L1209))!